src/actionable_entities.ts

```diff
@@ -0,0 +1,21 @@
+import { ActionableEntity } from "./actionable_entity";
+import { Sign } from "./sign";
+import { Sprite } from "./sprite";
+import { Point } from "./point";
+import { angry_dialog, sample_dialog, glitchy_dialog } from "./messages/dialog_graph";
+/*
+export function get_actionable_entities(canvas_w?: number, canvas_h?: number): ActionableEntity[] {
+	return [
+		new Sign("angry_dialog", new Sprite(0, 0, 29, 31), new Point(canvas_w / 2 - 15, canvas_h / 2 - 100), 29, 31, true, 0, 1, null, 0.5, angry_dialog),
+		new Sign("sample_dialog", new Sprite(0, 0, 29, 31), new Point(canvas_w / 2 - 15, canvas_h / 2 - 100), 29, 31, true, 0, 1, null, 0.5, sample_dialog),
+		new Sign("glitchy_dialog", new Sprite(0, 0, 29, 31), new Point(canvas_w / 2 - 15, canvas_h / 2 - 100), 29, 31, true, 0, 1, null, 0.5, glitchy_dialog)
+	]
+}*/
+
+export function get_actionable_entities(canvas_w?: number, canvas_h?: number): ActionableEntity[] {
+	return [
+		new Sign("angry_dialog", new Sprite(0, 0, 29, 31), null, 29, 31, true, 0, 1, null, 0.5, angry_dialog),
+		new Sign("sample_dialog", new Sprite(0, 0, 29, 31), null, 29, 31, true, 0, 1, null, 0.5, sample_dialog),
+		new Sign("glitchy_dialog", new Sprite(0, 0, 29, 31), null, 29, 31, true, 0, 1, null, 0.5, glitchy_dialog)
+	]
+}
```

Comment:
Les paramètres ne sont pas utilisés : pas la peine d'avoir une fonction pour ça

```suggestion
export const actionable_entities = [
	new Sign("angry_dialog", new Sprite(0, 0, 29, 31), null, 29, 31, true, 0, 1, null, 0.5, angry_dialog),
	new Sign("sample_dialog", new Sprite(0, 0, 29, 31), null, 29, 31, true, 0, 1, null, 0.5, sample_dialog),
	new Sign("glitchy_dialog", new Sprite(0, 0, 29, 31), null, 29, 31, true, 0, 1, null, 0.5, glitchy_dialog)
];
```

Reply:
Je les utilisais à un moment, d'où la fonction, je réfléchis encore à savoir si on va en avoir besoin ou pas. Pour centrer certains elements sur la carte par exemple.